xray/commands/curation/audit.go

```diff
@@ -169,13 +169,13 @@ func (ca *CurationAuditCommand) Run() (err error) {
 			err = errors.Join(err, e)
 		}
 	}
-	if ca.Progress != nil {
-		if e := ca.Progress.Quit(); e != nil {
+	if ca.Progress() != nil {
+		if e := ca.Progress().Quit(); e != nil {
 			err = errors.Join(err, e)
 		}
```

Comment:
`errors.Join` handles nil values very nicely, so tha above can be simplified to - 
```
err = errors.Join(err, ca.Progress().Quit())
```
The same goes for line 178.

Reply:
done